package.json

```diff
@@ -76,7 +76,8 @@
 		"patchedDependencies": {
 			"@wordpress/edit-site@5.15.0": "bin/patches/@wordpress__edit-site@5.15.0.patch",
 			"@wordpress/env@10.17.0": "bin/patches/@wordpress__env@10.17.0.patch",
-			"@wordpress/data@10.0.2": "bin/patches/@wordpress__data@10.0.2.patch"
+			"@wordpress/data@10.0.2": "bin/patches/@wordpress__data@10.0.2.patch",
+			"@wordpress/e2e-test-utils-playwright@1.19.1": "patches/@wordpress__e2e-test-utils-playwright@1.19.1.patch"
```

Comment:
Should be placed under `bin/patches/` directory, along with other patches.

Reply:
Fixed with 2e0ae4d21da9101f723e223e2e894797e663e69b